PASERK/k2.secret.json

```diff
@@ -3,21 +3,21 @@
   "tests": [
     {
       "name": "k2.secret-1",
-      "key": "00000000000000000000000000000000000000000000000000000000000000003b6a27bcceb6a42d62a3a8d02a6f0d73653215771de243a63ac048a18b59da29",
+      "key": "3b6a27bcceb6a42d62a3a8d02a6f0d73653215771de243a63ac048a18b59da29",
```

Comment:
Your PR is incorrect: You're truncating the secret key and leaving only the public key in the "key" field. The secret key is the "seed" attribute.

Sorry for not noticing this was a PR rather than an issue at first.

Reply:
First of all, I want to say that I like PASETO and I don't mean to put it down at all.

According to the RFC, the length of a private key is 32 bytes, which I think is common sense. Since the public key can be derived from this 32-byte long private key, there is no point in adding this to the payload of k*.secret. It is too long.

I don't care about the interpretation of "key" or "seed" in the test vector, the resulting "paserk" is the problem and I don't think we should include the public key here.

Reply:
Regardless of what any RFC says, 32-byte secret keys will [break every libsodium-based implementation](https://3v4l.org/bBM3B). PASERK is a PASETO extension and is not required to be implemented.

Reply:
I want to implement PASERK... 😢

Reply:
The `k4.secret` PASERK type contains all the information you need to implement it. Yes, it includes extraneous information (i.e. the public key) that your specific system doesn't need. However, many systems will need it. To better support interoperability, we're going to always include it. Feel free to slice it down to 32 bytes in your application after decoding.

Reply:
If you feel this behavior is incorrect, open a ticket with https://github.com/jedisct1/libsodium to support RFC-compatible 32-byte keys.

Reply:
OK. I'm not convinced, but I'll follow the current specifications for now. 

However, I would like to say that the key format of libsodium is special. And I don't think a specification that depends on one implementation will work in the IETF.